tests/spawner/test_spawner.py

```diff
@@ -36,14 +36,14 @@ def setup_method(self):
             schedule_type="TrainSchedule",
             strategy_type="RegularScheduleStrategy",
             train_schedule_train_type=self.TRAIN_TYPES[0],
-            regular_strategy_start_tick=self.START_TICKS[0],
+            strategy_start_tick=self.START_TICKS[0],
```

Comment:
Also here you may want to also use end_tick. Or maybe I don't get why that is not what you want, that is also totaly possible.

Reply:
See above.

As `end_tick` is tested in both strategies it is not necessary here in my opinion. Contrary setting it to None tests that it also works when `end_tick` is None.